perfdash/config.go

```diff
@@ -80,6 +81,21 @@ var (
 				OutputFilePrefix: "ResourceUsageSummary",
 				Parser:           parseResourceUsageData,
 			}},
+			"LoadHighThroughputStartup": []TestDescription{{
+				Name:             "load",
+				OutputFilePrefix: "PodStartupLatency_HighThroughputStartupLatency",
+				Parser:           parsePerfData,
+			}},
+			"LoadPodStartup": []TestDescription{{
+				Name:             "load",
+				OutputFilePrefix: "PodStartupLatency_PodStartupLatency",
+				Parser:           parsePerfData,
+			}},
+			"LoadSaturationPodStartup": []TestDescription{{
```

Comment:
We don't have such measurement anywhere (saturation changed name to high-throughput)

Reply:
This measurement exists in density test - I've copied the name in case if someone will enable the same measurement in load test.

Changed `Saturation` to `Second` to match Identifier from `experimental-config.yaml`.